### DISCUSSION TITLE
Release 0.4.0 is planned for Q3 2024

### DISCUSSION THREAD

**Discussion by JohnChe88** [aws-samples/spark-on-aws-lambda#61] [Announcements]
We are currently planning,
 
1. Upgrade Python base image
2. Upgrade spark version to 3.5.1
3. Upgrade the Iceberg version
4. Benchmark around AWS lambda initialization

**Comment by JohnChe88** [comment]
The Initialization duration is now ~ 550ms

**Reply by vivi609kfes** [reply]
Hellooo @JohnChe88 
when can we see the newest version?
thanks~~ 
Because I need to change my Python version, I was keep failing .

**Reply by JohnChe88** [reply]
@vivi609kfes Release v0.4.0
